components/Card/index.js

```diff
@@ -12,7 +13,11 @@ export default function PlantCard({ onToggleFavorite, isFavorite, plant }) {
         isFavorite={isFavorite}
       />
       <StyledLink href={`/plants/${plant?.id}`}>
-        <StyledFigure>
+        <StyledFigure $categoryColor={  
+            categories.find(  
+              (category) => category.slug === plant.categorySlug  
+            ).bgcolor  
+          }>
```

Comment:
computations like this are often better placed above the return in a separate const.

Makes it easier to read and understand.

```jsx
const categoryColor =  categories.find((category) => category.slug === plant.categorySlug).bgcolor

return <StyledFigure $categoryColor={categoryColor}/> 
```